tools/reactive-controllers/roving-tab-index.md

```diff
@@ -43,50 +49,440 @@ class Container extends LitElement {
+    <div role="menuitem">New</div>
+    <div role="menuitem">Open</div>
+</div>
+```
+
+</sp-tab-panel>
+</sp-tabs>
+
+#### Keyboard support
+
+The `RovingTabindexController` provides the following keyboard interactions:
+
+<sp-table>
+    <sp-table-head>
+        <sp-table-head-cell>Key</sp-table-head-cell>
+        <sp-table-head-cell>Direction Mode</sp-table-head-cell>
+        <sp-table-head-cell>Action</sp-table-head-cell>
+    </sp-table-head>
+    <sp-table-body>
+        <sp-table-row>
+            <sp-table-cell><strong>Tab</strong></sp-table-cell>
+            <sp-table-cell>All</sp-table-cell>
+            <sp-table-cell>Moves focus into or out of the composite widget</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell><strong>→ (Right Arrow)</strong></sp-table-cell>
+            <sp-table-cell>horizontal, both, grid</sp-table-cell>
+            <sp-table-cell>Moves focus to the next element</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell><strong>← (Left Arrow)</strong></sp-table-cell>
+            <sp-table-cell>horizontal, both, grid</sp-table-cell>
+            <sp-table-cell>Moves focus to the previous element</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell><strong>↓ (Down Arrow)</strong></sp-table-cell>
+            <sp-table-cell>vertical, both, grid</sp-table-cell>
+            <sp-table-cell>Moves focus to the next element (or down in grid)</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell><strong>↑ (Up Arrow)</strong></sp-table-cell>
+            <sp-table-cell>vertical, both, grid</sp-table-cell>
+            <sp-table-cell>Moves focus to the previous element (or up in grid)</sp-table-cell>
+        </sp-table-row>
+        
+    </sp-table-body>
+</sp-table>
+
+#### Disabled elements
+
+Use the `isFocusableElement` option to skip disabled elements:
+
+```typescript
+rovingTabindexController = new RovingTabindexController<Button>(this, {
+    elements: () => [...this.querySelectorAll('sp-button')],
+    isFocusableElement: (button) => !button.disabled,
+});
+```
+
+Ensure disabled elements have appropriate ARIA attributes:
+
+```html
+<sp-button disabled aria-disabled="true">Disabled</sp-button>
+```
+
+#### Screen reader announcements
+
+When selection changes, announce it to screen readers:
+
+```typescript
+elementEnterAction: (element) => {
+    this.selected = element;
+
+    // Update aria-selected
+    this.elements.forEach((el) => {
+        el.setAttribute('aria-selected', String(el === element));
+    });
+
+    // Optional: Announce change
+    const announcement = document.createElement('div');
+    announcement.setAttribute('role', 'status');
+    announcement.setAttribute('aria-live', 'polite');
+    announcement.textContent = `Selected: ${element.textContent}`;
+    // ... add to DOM temporarily
+};
+```
```

Comment:
This selection can be removed. The screenreader will announce the change in selection based on the change in ARIA state. For example, when a tab is `aria-selected` the screenreader will automatically anounce.
```suggestion
```